errors/src/commonMain/kotlin/dev/icerock/moko/errors/registry/ConditionPair.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.errors.registry
+
+data class ConditionPair(
```

Comment:
It can't be internal, because it is used in the public inline fucntion: https://github.com/icerockdev/moko-errors/pull/1/files#diff-a39f3461125c87cdfd9867692fffcea6R71